src/configuration/cfg-provider.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Set up the default interface
+ip link set dev eth0 up
+ip addr add 10.0.0.89/24 dev eth0
```

Comment:
```suggestion
ip addr add 10.0.0.1/24 dev eth0
```
Any reason we cannot go with the first address in the subnet? Perhaps we should even limit the network to a `/30` to make it clear that this is, in effect, a point-to-point connection?